### PR TITLE
build: Release 3.11.0

### PR DESCRIPTION
**Pull request by creyD**
## Related resources

* #...
* #...

## Checklist

* [ ] I have opened this pull request against ``develop``
* [x] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on [Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

File: cms/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import django
 
-__version__ = '3.10.0'
+__version__ = 'test'
```
Comment (marksweb) [inline cms/__init__.py:3]: This is my fault. I've broken the release script where this is bumped 🤦 

Fix for this here; https://github.com/django-cms/django-cms/pull/7322

**Review by marksweb — CHANGES_REQUESTED** [review]
The static files haven't been given the updated version number. I'm going to assume this is because of the bug I caused to bumping the version number.

File: cms/__init__.py (hunk 1 above)
Comment (creyD) [reply cms/__init__.py:3]: Should I run the script again, or would it be enough if I just search and replaced the 3.10.0?

Comment (marksweb) [reply cms/__init__.py:3]: @creyD Yeah, my fix is merged so I think we'll close this down, delete the release branch from here, you can do the same from your fork. Then run it again and we should see this version as `3.11.0` and the static files bundle in a folder matching the version.

**Pull request closed without merging**